libr/bin/dwarf.c

```diff
@@ -384,6 +385,10 @@ static RBinSection *getsection(RBin *bin, int sn) {
 		}
 		r_list_foreach (o->sections, iter, section) {
 			if (strstr (section->name, name_str)) {
+				if (strncmp (section->name, ".debug_", 7) == 0 && R_BIN_ELF_SCN_IS_COMPRESSED(section->flags))  {
```

Comment:
use r_str_startswith and add missing space before (

Reply:
Done